packages/less/tsconfig.json

```diff
@@ -9,6 +9,11 @@
     "importHelpers": true,
     "target": "ES5"
   },
+  "ts-node": {
```

Comment:
Add the special ENV for CI , https://github.com/TypeStrong/ts-node/issues/693#issuecomment-848948517 for more details.